lib/internal/modules/cjs/loader.js

```diff
@@ -824,12 +825,30 @@ Module.runMain = function() {
   Module._load(process.argv[1], null, true);
 };
 
-Module.createRequireFromPath = (filename) => {
+function createRequireFromPath(filename) {
   const m = new Module(filename);
   m.filename = filename;
   m.paths = Module._nodeModulePaths(path.dirname(filename));
   return makeRequireFunction(m);
-};
+}
+
+Module.createRequireFromPath = createRequireFromPath;
+
+function createRequire(filename) {
+  let path;
+  if (typeof filename === 'string' && filename.startsWith('.')) {
+    throw new ERR_INVALID_CREATE_REQUIRE_PATH(filename);
+  } else if (typeof filename === 'object' && filename instanceof URL ||
+    typeof filename === 'string' && !filename.startsWith('/')) {
```

Comment:
`filename.startsWith('/')` likely won't work on Windows? `path.isAbsolute` may be worth using here as it explicitly does the Windows drive letter detection, distinguishing drive letters from file URLs.

Alternatively we could do the inverse - `filename.startsWith('file:')`